Gems/Atom/RHI/Vulkan/Code/Source/RHI/DescriptorSet.h

```diff
@@ -73,6 +73,9 @@ namespace AZ
             RHI::Ptr<BufferView> GetConstantDataBufferView() const;
 
         private:
+            // we use the small_vector to speed up SRG compilation, but we have to be a bit careful with the size of the pre-allocation:
+            // e.g. the StarterGame - Level has about 8000 objects (~4000 meshes with 2 LODs) that get a unique Draw-Srgs for
+            // each pass. And with 2 Material-Pipelines we have about 20 passes, so we end up with 160.000 unique Draw-Srgs
```

Comment:
What is in DrawSRG? This could easily be solved with Bindless but we still need to support non-bindless workflow

Reply:
At a quick glance the DefaultDrawSrg doesn't contain anything that couldn't be part of the ObjectSrg generally, but i also see two things that won't work out of the box:
- uvStreamTangentBitMask: This depends on the Shader Input Contract somehow, so it can be different for each draw call. But i also don't see why the information shouldn't stay consistent for a mesh.
- Shader-Options, or, at least the `ShaderVariantFallback = 128;`: Those are again different for each draw call, but with specialization constants those shouldn't be relevant. Not sure if they are simply missing from the layout then...